assets/src/views/flight-request.js

```diff
@@ -0,0 +1,28 @@
+const ko = require('knockout');
+import { getFlightPrice, getDiscount } from "../libs/flight-utils";
+
+
+function FlightRequestViewModel(method) {
+  this.budget = ko.observable($("#id_budget").val());
+  this.regions = ko.observable([...document.querySelectorAll("form [name='regions']:checked")].map(function (node) { return node.value }));
+  this.topics = ko.observable([...document.querySelectorAll("form [name='topics']:checked")].map(function (node) { return node.value }));
+
+  this.pricing = JSON.parse($('#data-pricing').text());
```

Comment:
Is this used? Seems like we're loading pricing from there in `getFlightPrice` also?

Reply:
No. It can be removed.